### ISSUE TITLE
Ignore App Config file on Update

### ISSUE THREAD

**Issue by davidbuckleyni** [Squirrel/Squirrel.Windows#1499]
I have an app that is on six pcs that has a config setting unique to them I want to keep that setting intact when the applicaiton installs or updates theirs an existing version their at the min how do i acheive this and also how to I give it a target destiniation for the folder To Install to

**Comment by gojanpaolo** [comment]
Hello @davidbuckleyni, I suggest you persist the user settings on a separate directory outside of the app's install location. See also #921

Install location can't be changed as mentioned in #1002

**Comment by davidbuckleyni** [comment]
Ok that is fine and understandable thanks great piece of software esp the incremental updates.

**Comment by gojanpaolo** [comment]
I agree! :) please close the issue if you don't have any more questions. Thankds!
